src/distributors/adoptium/adoptium-installer.ts

```diff
@@ -65,18 +65,18 @@ export class AdoptOpenJDKDistributor extends JavaBase {
     javaPath = await tc.cacheDir(
       archivePath,
       this.toolcacheFolderName,
-      javaRelease.resolvedVersion,
+      javaRelease.version,
       this.architecture
     );
 
     if (process.platform === 'darwin') {
       javaPath = path.join(javaPath, macOSJavaContentDir);
     }
 
-    return { javaPath, javaVersion: javaRelease.resolvedVersion };
+    return { javaPath, javaVersion: javaRelease.url };
```

Comment:
I think `javaRelease.version` should be here.

Reply:
Done. Thank you.